srcpkgs/virtio_vmmci/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'virtio_vmmci'
+pkgname=virtio_vmmci
+version=0.3.0
+revision=1
+depends="dkms"
+short_desc="Virtio driver for OpenBSD VMM Control Interface"
+maintainer="Renato Aguiar <renato@renatoaguiar.net>"
+license="GPL-2.0-only"
+homepage="https://github.com/voutilad/virtio_vmmci"
+distfiles="https://github.com/voutilad/virtio_vmmci/archive/${version}.tar.gz"
+checksum=ea8d952a8989d3a107717cb63385450bc4cb07f805ba8a61432ea352d2eb0229
+dkms_modules="virtio_vmmci ${version}"
+
+do_install() {
+	vmkdir usr/src/virtio_vmmci-${version}
+	vcopy "*.c" usr/src/virtio_vmmci-${version}
+	vcopy "*.h" usr/src/virtio_vmmci-${version}
+	vcopy Makefile usr/src/virtio_vmmci-${version}
+	vcopy Kconfig usr/src/virtio_vmmci-${version}
+	vcopy Kbuild usr/src/virtio_vmmci-${version}
+	vinstall ${FILESDIR}/dkms.conf 644 usr/src/virtio_vmmci-${version}
```

Comment:
They have no makefile at all to install this stuff? :(

Do you think development is likely to continue?